libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -20,15 +21,78 @@ bool _ebpf_platform_is_preemptible = true;
 
 extern "C" bool ebpf_fuzzing_enabled = false;
 
+// Thread pool related globals.
+static TP_CALLBACK_ENVIRON _callback_environment;
+static PTP_POOL _pool = NULL;
+static PTP_CLEANUP_GROUP _cleanup_group = NULL;
```

Comment:
```suggestion
static PTP_POOL _pool = nullptr;
static PTP_CLEANUP_GROUP _cleanup_group = nullptr;
```
nit: use C++ keyword nullptr in .cpp files wherever possible (same in line 37, 51, etc.)

Reply:
Fixed all usage of `NULL` in this file.